go-controller/pkg/network-attach-def-controller/network_attach_def_controller.go

```diff
@@ -169,8 +170,12 @@ func (nadController *NetAttachDefinitionController) syncAll() (err error) {
 			continue
 		}
 		err = nadController.syncNAD(key, nad)
-		if err != nil {
-			return fmt.Errorf("%s: failed to sync %s: %v", nadController.name, key, err)
+		if err != nil && !errors.Is(err, config.ErrorAttachDefNotOvnManaged) {
+			if nadController.recorder != nil {
+				nadController.recorder.Eventf(&corev1.ObjectReference{Kind: nad.Kind, Namespace: nad.Namespace, Name: nad.Name}, corev1.EventTypeWarning,
+					"FailedSync", "Failed to sync network %s config: %v", key, err)
+			}
+			klog.Errorf("%s: failed to sync %s: %v", nadController.name, key, err)
```

Comment:
Is this an error or warning? like ovnk shouldn't be managing these anyways right? its a warning to the user they did something wrong? or a proper error happened for ovnk?

Reply:
> its a warning to the user they did something wrong?

Correct. Today, thats all that can cause this, so I agree, Ill change it to Warning. 
Warning event is the highest "error" event level.